src/commands/serve.ts

```diff
@@ -73,24 +64,35 @@ export const builder = (yargs: Argv) =>
       bootstrapPerspective: {
         type: 'string',
         describe: 'Path to Bootstrap perspectives json file (list of perspectives)'
+      },
+      appLangAliases: {
+        type: 'string',
+        describe: 'Language aliases to be loaded into ad4m-executor',
+        default: '{}'
       }
     });
 
 export const handler = async (argv: Arguments<Options>): Promise<void> => {
-  const { port, hcAdminPort, hcAppPort, connectHolochain, dataPath, defaultLangPath, networkBootstrapSeed, languageLanguageOnly, bootstrapLanguage, bootstrapPerspective } = argv;
+  const { port, hcAdminPort, hcAppPort, connectHolochain, languageLanguageOnly, dataPath, bootstrapLanguage, bootstrapPerspective, appLangAliases } = argv;
 
-  const binaryPath = path.join(getAppDataPath(dataPath || 'ad4m'), 'binary');
+  const globalConfig = getConfig();
 
-  const builtInlang = defaultLangPath as string;
+  if(!globalConfig[dataPath || '']) {
+    throw Error('No config found, please run ad4m-host init with the dataPath & networkBootstrapSeed params')
+  }
 
-  const appDefaultLangLocation: string =  path.isAbsolute(builtInlang) ? builtInlang : path.join(__dirname, builtInlang);
+  const { seedPath } = globalConfig[dataPath || ''];
 
-  const gqlPort = await getPort({ port })
+  const binaryPath = path.join(getAppDataPath(dataPath || 'ad4m'), 'binary');
```

Comment:
When `dataPath` is provided to the  `init` & `serve` functions, should we just use the `dataPath` directly, vs passing into `getAppDataPath()` since this is supposed to be a relative path to the data directory, and not one which would originate in the appDataPath?

Reply:
`dataPath` is the folder name should we maybe change it to something else like `environment` or something else

Reply:
Ah okay I see, I guess we should also change the description from: The relative path for storing ad4m data, to something like: Name of directory to store ad4m data in within the systems app data path.